docker/namada-wasm/Dockerfile

```diff
@@ -1,18 +1,16 @@
 # This docker is used for deterministic wasm builds
 
 # The version should be matching the version set in wasm/rust-toolchain.toml
-FROM rust:1.61.0
+FROM rust:1.61
```

Comment:
should stay 1.61.0; right now 1.61 is an alias for it but it might have a patch release

Reply:
yeah, lets keep `rust:1.61.0`. I'll make a commit